packages/logging/src/__tests__/environment.test.ts

```
@@ -0,0 +1,186 @@
+  isProduction,
+  isTest,
+  parseLogLevel,
+} from '../environment.js'
+import { LogLevel } from '../types.js'
+
+describe('Environment Detection', () => {
+  const originalNodeEnv = process.env.NODE_ENV
+  const originalLogLevel = process.env.LOG_LEVEL
+
+  beforeEach(() => {
+    delete process.env.NODE_ENV
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)